pyproject.toml

```diff
@@ -57,5 +57,8 @@ script-files = [
     "src/scripts/spkdeploy",
 ]
 
+# Ensure package data such as resources are included
+package-data = { "schnetpack.train" = ["ressources/partition_spline_for_robust_loss.npz"] }
```

Comment:
same here

Reply:
this is needed to include the file when building the package